app/getting-started-guide/2.1.x/prepare.md

```diff
@@ -13,30 +13,100 @@ Before you start this section, make sure that:
 
 In this guide, an instance of {{site.base_gateway}} is referenced via `<admin-hostname>`. Make sure to replace `<admin-hostname>` with the hostname of your control plane instance.
 
+## (Free Trials Only) Expose the Admin API {#free-trials-setup}
+
 <div class="alert alert-ee">
-<h5><img class="no-image-expand" src="/assets/images/icons/icn-enterprise-grey.svg" alt="Enterprise" />Note for {{site.ee_product_name}} free trial users</h5>
-If you are trying out {{site.ee_product_name}} using a hosted free trial, you can go through this guide to experience some of this enhanced functionality. However, you need to enable access to the Admin API first.
-  <ol>
-    <li>Open the Kong Manager using the link provided in your free trial welcome email.</li>
-    <li>Open <strong>Teams</strong> > <strong>RBAC Users</strong>, then click <strong>Add New User</strong>.</li>
-    <li>Set up an RBAC user with an appropriate role (e.g. admin) and an RBAC token. Make sure to leave the <strong>Enabled</strong> box checked.</li>
-    <li>Test by building a request to the Admin API.
-<p></p>
-<span style="font-size:14px"><em>Using cURL:</em></span>
-<pre class="highlight">
-<code>$ curl -H “Kong-Admin-Token:&lt;your-RBAC-token&gt;” \
-&lt;admin-endpoint-from-email&gt;/services</code></pre>
-<span style="font-size:14px"><em>Or using HTTPie:</em></span>
-<pre class="highlight">
-<code>$ http &lt;admin-endpoint-from-email&gt;/services \
-Kong-Admin-Token:&lt;your-RBAC-token&gt; </code></pre>
-You should get a 200 response code and your list of services will be empty.
-<br/>
-<br/>You can now use the Admin API instructions in this guide. Remember to switch any mentions of <code>http://&lt;admin-hostname&gt;:8001</code> or <code>&lt;admin-hostname&gt;:8001</code> with your Admin API endpoint (no port number required).
-</li>
-</ol>
+  <img class="no-image-expand" src="/assets/images/icons/icn-enterprise-grey.svg"
+  alt="Enterprise" />
+  This section is only applicable to {{site.ee_product_name}} hosted free trials.
+  Skip to <a href="#verify-the-kong-gateway-configuration">Verify the Kong Gateway Configuration</a>
+  if you aren't a free trial user.
 </div>
 
+If you are trying out {{site.ee_product_name}} using a hosted free trial, you
+can go through this guide to experience some of this enhanced functionality.
+However, if you want to test out configuring {{site.ee_product_name}}
```

Comment:
We recommend that you get started using the Kong Manager user interface; however, if you want to test out configuring {{site.ee_product_name}}